### PR TITLE
[minor_change] Add custom_epg_name attribute to resource "aci_epg_to_domain"

### PR DESCRIPTION
**Pull request by shrsr**

### CODE REVIEW

**Review by lhercot — CHANGES_REQUESTED** [review]
Update example and documentation

File: go.sum [hunk 1]
```diff
@@ -67,6 +67,8 @@ github.com/chzyer/readline v0.0.0-20180603132655-2972be24d48e/go.mod h1:nSuG5e5P
 github.com/chzyer/test v0.0.0-20180213035817-a1ea475d72b1/go.mod h1:Q3SI9o4m/ZMnBNeIyt5eFwwo7qiLfzFZmjNmxjkiQlU=
 github.com/ciscoecosystem/aci-go-client v1.31.0 h1:0zQV68FtlrlWG4ArnMFKnienBizpZatv548HJQe5Wsk=
 github.com/ciscoecosystem/aci-go-client v1.31.0/go.mod h1:pGK3tDhR/D4+P5f6utT1V7TQQxDw2fL7j79QXeVk6po=
+github.com/ciscoecosystem/aci-go-client v1.33.0 h1:/DegzYxTGQrA0C9zMVN50OKqSKW4RkjrQbzP8jK4mFI=
```
Comment (xinyuezhao) [inline go.sum:70]: you may want to remove aci-go-client v1.31.0

**Review by xinyuezhao — DISMISSED** [review]

File: website/docs/r/epg_to_domain.html.markdown [hunk 2]
```diff
@@ -50,6 +51,7 @@ resource "aci_epg_to_domain" "example" {
 Allowed values: "none", "staticBinding", "dynamicBinding", "ephemeral". Default value: "none"
 * `allow_micro_seg` - (Optional) Boolean flag for allow micro segment. default value will be "false".
 "true" maps to class_pref="useg" and "false maps to class_pref="encap" 
+* `custom_epg_name` - (Optional) Custom Epg name for object Domain.
```
Comment (lhercot) [inline website/docs/r/epg_to_domain.html.markdown:54]: Custom EPG name used as name of the VMM port group for the domain.

**Review by lhercot — CHANGES_REQUESTED** [review]
Seems the new vendor files are missing. Can you make sure you do `go mod vendor` and commit the changed files?

**Review by anvitha-jain — COMMENTED** [review]
Everywhere in resource and datasource files, where  -> ("rsdomAtt-[%s]", tDn) is present shouldn't it be replaced by thr Rnattribute?

**Comment by shrsr** [comment]
@anvitha-jain For this particular resource we don't have RN defined as a constant on SDK, so it should remain ("rsdomAtt-[%s]", tDn)

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**